specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerPendingConnectionList.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "List Pending Azure Virtual Network Manager Scope Connection",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subscriptionC",
+    "resourceGroupName": "rg1",
+    "networkManagerName": "testNetworkManager"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "ETag": "be3cfaa2-3d02-4624-92d7-a26766306c8b"
+      },
+      "body": {
+
+        "value": [
+          {
+            "id": "/subscriptions/subscriptionA/providers/Microsoft.Network/networkManagerConnections/TestNMConnection",
+            "type": "Microsoft.Network/networkManagerConnections",
+            "name": "TestNMConnection",
+            "systemData": {
+              "createdBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+              "createdByType": "User",
+              "createdAt": "2021-01-11T18:52:27Z",
+              "lastModifiedBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+              "lastModifiedByType": "User",
+              "lastModifiedAt": "2021-01-11T18:52:27Z"
+            },
+            "properties": {
+              "networkManagerId": "/subscriptions/subscriptionC/resourceGroup/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager",
+              "connectionState": "Pending"
+            }
+          }
+        ],
+        "nextLink": "{baseurl}//subscriptions/subscriptionC/resourceGroups/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/listPendingNetworkManagerConnections?api-version=2021-02-01-preview&$skipToken=10"
```

Comment:
>listPendingNetworkManagerConnections [](http://example.com/codeflow?start=142&length=36)

Is this correct ? This seems to be defined as a POST call . .but the next link would be a GET

Reply:
Correct. The listPending should be a Post call to list which returns a skipToken for subsequent pages, not a nextLink. Will update